app/tests/unit/components/Attachment/AttachmentTableRow.test.tsx

```diff
@@ -59,15 +55,8 @@ const componentTestingHelper =
     getPropsFromTestQuery: (data) => ({ attachment: data.query.attachment }),
   });
 
-const routerPush = jest.fn();
-
 describe("The Attachment table row component", () => {
   beforeEach(() => {
-    jest.resetAllMocks();
```

Comment:
Do we want to remove this? The jest config clears all mocks automatically, but it doesn't like there's anything automatic for resetting mocks